backend/foodbank_southlondon/api/views.py

```diff
@@ -8,10 +8,14 @@
 # CONFIG VARIABLES
 _FBSL_COLLECTION_CENTRES = "FBSL_COLLECTION_CENTRES"
 
+calendar = rest.model("Calendar", {
```

Comment:
It felt natural to model this as an object now there's more than one field but I'm not fussed either way, it could just be two lists like how we end up passing in the embed URL anyway.

This change isn't backwards compatible with old SPA builds but I don't think that matters for us generally? Especially since this feature isn't used yet.